megazords/ios/RustAppServices.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,684 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 51;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		C852EE9E220A283200A6E79A /* libsqlcipher.a in Frameworks */ = {isa = PBXBuildFile; fileRef = C852EE9D220A283200A6E79A /* libsqlcipher.a */; };
+		C852EED3220A29FE00A6E79A /* String+Free_Logins.swift in Sources */ = {isa = PBXBuildFile; fileRef = C852EECA220A29FE00A6E79A /* String+Free_Logins.swift */; };
+		C852EED5220A29FE00A6E79A /* LoginRecord.swift in Sources */ = {isa = PBXBuildFile; fileRef = C852EECC220A29FE00A6E79A /* LoginRecord.swift */; };
+		C852EED6220A29FE00A6E79A /* LoginsStorage.swift in Sources */ = {isa = PBXBuildFile; fileRef = C852EECD220A29FE00A6E79A /* LoginsStorage.swift */; };
+		C852EED7220A29FE00A6E79A /* LoginStoreError.swift in Sources */ = {isa = PBXBuildFile; fileRef = C852EECF220A29FE00A6E79A /* LoginStoreError.swift */; };
+		C852EED8220A29FE00A6E79A /* LockError.swift in Sources */ = {isa = PBXBuildFile; fileRef = C852EED0220A29FE00A6E79A /* LockError.swift */; };
+		C852EEE6220A2A2B00A6E79A /* RustPointer.swift in Sources */ = {isa = PBXBuildFile; fileRef = C852EEDC220A2A2B00A6E79A /* RustPointer.swift */; };
+		C852EEE7220A2A2B00A6E79A /* FirefoxAccount.swift in Sources */ = {isa = PBXBuildFile; fileRef = C852EEDD220A2A2B00A6E79A /* FirefoxAccount.swift */; };
```

Comment:
I'm a bit concerned by the fact that we have to re-list `.swift` files in the megazord project, on top of listing them in the original projects. Is there any way to go around this without listing explicitly these source files?

Reply:
I don't think so, I remember xcode and visual studio not supporting globbing or similar is one reason CMake is so wonky in practice about file lists.

Reply:
Correct no globbing. I could avoid the duplicate file lists and have the FxAClient and Logins build additional static lib targets, and then the megazord would include those two .a files (include FxAClient and Logins as Xcode sub-projects in the megazord Xcode project).

The only problem is the SerialQueue.swift. Right now that file would be included twice.
My understanding of that queue is that there should be only one instance of it, even if the consumer includes multiple app services frameworks. Since the queue name is deterministic, I think an acquire-or-init getter for the queue will work ok. /me looks into it...

Reply:
`DispatchQueue(somename)` is a non-failable init, but I am not clear what happens when that is called multiple times, because that init is also setting up various params for the queue, for instance it could be changed from a serial to concurrent queue. What happens to the previous references to this queue I wonder. I'll have to experiment and see.

Reply:
Answer: creates separate queues.